lasagne/utils.py

```diff
@@ -305,30 +306,50 @@ def create_param(spec, shape, name=None):
     name : string, optional
         If a new variable is created, the name to give to the parameter
         variable. This is ignored if `spec` is already a Theano expression
-        or shared variable.
+        or shared variable and which already has a name.
```

Comment:
Needs to be changed a bit because it doesn't only apply when a new variable is created now:
```
The name to give to the parameter variable. Ignored if `spec` is or returns a Theano expression or shared variable that already has a name.
```